src/components/DropDownText/DropDownText.tsx

```diff
@@ -0,0 +1,50 @@
+import { Button, Typography } from "@mui/material"
+import React, { useState } from 'react'
+import RenderIf from "../RenderIf/RenderIf"
+import { colors } from '../../constants/constants'
+
+interface DropDownTextProps {
+  key: string
+  textObject: {
+    title: string
+    detail: string
+  }}
+
+const DropDownText = (props: DropDownTextProps) => {
+  const [ isHovered , setIsHovered ] = useState(false)
+  const { textObject: { title, detail } } = props
+
+  return (
+    <>
+      <Button
+        onMouseOver={()=> setIsHovered(true)}
+        onMouseLeave={()=> setIsHovered(false)}
+        sx={{color: colors.secondary }}
+        >
+        <Typography
+          variant="h6"
+          sx={{
```

Comment:
Generally I find it's much cleaner to break out styles from the inline. These often bloat this file without offering much benefit of being visible here. 

When I'm in a file reading I care about the logic that is happening here, the states, the stores, to functions. I don't typically care about styles.